cloudbuild.yaml

```diff
@@ -27,6 +27,7 @@ steps:
            'mapaction-develop-mvp',
            '--location', 'europe-west2',
            '--update-env-variables=AIRFLOW_VAR_DOCKER_IMAGE_VERSION=$COMMIT_SHA']
+timeout: 900s
```

Comment:
I think this needs to be a string (`'900s'`)

Reply:
The example here is not quoted as a string: https://cloud.google.com/build/docs/build-config#timeout_2
I'll try it and see what happens.

Reply:
Ah good point. Sorry, I should have expanded my reasoning: [here](https://cloud.google.com/build/docs/build-config) it says that `timeout` should be a string, but I think it will anyway be parsed as a string even without the quotes